tb_plugin/README.md

```diff
@@ -41,7 +41,178 @@
 
   Open URL `http://localhost:6006` in the browser.
 
-* Navigate to TORCH_PROFILER tab
+* Navigate to PYTORCH_PROFILER tab
 
-  If the files under `--logdir` are too big or too many.
-  Refresh the browser to check latest loaded result.
+  If the files under `--logdir` are too big or too many, 
+  please wait a while and refresh the browser to check latest loaded result.
+
+### Quick Usage Instructions
+
+We regard each running with profiler enabled as a "run".
+In most cases a run is a single process. If DDP is enabled, then a run includes multiple processes.
+We name each process as a "worker".
+
+Each run corresponds to a sub-folder under "--logdir" specified folder.
+Under each run's sub-folder, each file is one worker's dumped chrome tracing file.
```

Comment:
Because a sub-folder could contains multiple files corresponding to multiple processes, what about update to "Each sub-folder contains one or more chrome trace file, one for each process."?

Reply:
Yes, more clear, thanks!